src/_guides/language/analysis-options.md

```diff
@@ -121,19 +132,19 @@ analyzer:
 
```

Comment:
As I told you in person, these flags don't make sense to me. To enable both flags, you set them to false? And then there's the double-negative: `no-implicit-*: false`. So confusing.